data/MoNihal10.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name" : "Mohammed Nihal" ,
+  "displayStatsPublic" :true ,
+  "type" : "personal" ,
+  "bio" : " cs undergrad!!! " ,
+  "avatar" : "https://github.com/MoNihal10.png",
+  "links" :[
+  {
+    "name" : "Twitter" ,
+    "url" : "https://twitter.com/n_ihal_10",
+    "icon" : "FaTwitter"
+  },
+  {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/n_ihal_10",
+      "icon": "FaInstagram"
+    },
+   {
+      "name": "LinkedIn",
+      "url": "www.linkedin.com/in/mohammed-nihal-b80b6b227",
```

Comment:
```suggestion
      "url": "https://www.linkedin.com/in/mohammed-nihal-b80b6b227",
```

Reply:
the changes have been done.

Reply:
when is the custom web URL for my account going to work???

Reply:
> when is the custom web URL for my account going to work???

As soon as this is merged, a few minutes after that